data/card_catalogue/upgradeable-1.json

```diff
@@ -264,18 +104,18 @@
     "upgrade": "Inner Sun",
     "tier": "red",
     "details": {
-      "rules": "Deal 15 damage and set fire to all flammable non-player objects within 6 squares",
+      "rules": "Deal 15 damage and set fire to all flammable non-player objects within 6 squares.",
       "flavour": "The best part is that nice campfire smell.",
       "image": "assets/core/card/illustration/Immolate.png"
     }
   },
   {
-    "title": "Inner Sun",
+    "title": "Terrestrial Sun",
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Summon an Inner Sun the size of a baseball. The Inner sun is explosively flammable and burns forever.",
-      "flavour": "My rage burns with the force of 1000 suns.",
+      "rules": "Summon a brick of material which burns violently forever when set on fire.",
+      "flavour": "The fire in our hearts will never burn out",
       "image": "assets/core/card/illustration/Inner Sun.png"
     }
```

Comment:
Same effect, but I think the new wording makes a little more sense.

Reply:
I like this change, should it come pre-lit though?

Reply:
I think this makes it a little flexible, since you might want to throw it or put it somewhere before igniting it.

It shouldn't be too hard to light a fire, especially if they're a fire mage

Reply:
Sounds good, I'll be sure to give the players an idea of how violent we're talking in the illustration 👍

Reply:
Mentioned this in the issue for new illustrations https://github.com/elliottomlinson/rpcg/issues/105#issue-959796952